### DISCUSSION TITLE
config_ConfigureExports pipeline failing with API error

### DISCUSSION THREAD

**Discussion by wali12** [microsoft/finops-toolkit#827] [Q&A] [answered]
Hi,

I deployed the FinOps toolkit for the first time today and followed the steps in the documentation to setup.
When reviewing the pipeline run in the ADF, I saw that it failed with the following error:
![image](https://github.com/user-attachments/assets/ae340c79-d84f-43e2-b2e1-2a97b70cfb5c)

Confirmed the input does indeed have this combination. 
Any advice on how to resolve this?

**Comment by arthurclares** [comment]
@wali12 Is this a brand new deployment of 0.4? If yes, I will convert this into an issue.

**Comment by wali12** [comment]
Hi,

Yes. it is a brand new deployment. Looks like the recurrence and time frame parameters in config_ConfigureExports pipeline were mixed up.
One was TheLastMonth, Daily
and the other was MonthToDate, Monthly

I worked around the issue by manually editing the pipeline to adjust the parameters.

**Reply by arthurclares** [reply, marked as answer]
@wali12 I will open an Issue for this to keep track. Although, I believe @MSBrett  fixed this yesterday. #843
